src/storages/inRedis/SplitsCacheInRedis.ts

```diff
@@ -17,28 +18,29 @@ function processPipelineAnswer(results: Array<[Error | null, string]>): string[]
 }
 
 /**
- * Default ISplitsCacheSync implementation that stores split definitions in memory.
- * Supported by all JS runtimes.
+ * ISplitsCacheAsync implementation that stores split definitions in Redis.
+ * Supported by Node.
  */
 export default class SplitsCacheInRedis implements ISplitsCacheAsync {
 
   private readonly redis: Redis;
   private readonly keys: KeyBuilderSS;
-  private redisError?: string;
+  // private redisError?: string;
```

Comment:
If not used, remove this line.

Reply:
thanks ! will do

Reply:
actually, the code was uncommented again.